### PR TITLE
Uninstall page

### PR DESCRIPTION
**Pull request by kkosiorowska**
Refs [#86
](https://github.com/tahowallet/product-design/issues/86)
Refs [#1578
](https://github.com/tahowallet/extension/issues/1578)
This PR updates the last screen. Changes were made to styles and texts. A widget with a form was embedded inside the screen. Changes:

- add a typeform
- text updates
- add a gif 
- set a gradient for the background

## UI

Before
Screenshot 2023-03-14 at 15 23 04

After
Screenshot 2023-04-03 at 12 32 28

### CODE REVIEW

File: src/pages/goodbye.tsx [hunk 1]
```diff
@@ -76,17 +91,25 @@ function Goodbye() {
...
+      <div className="cover gradient-2" style={{ height }} />
+      <div className="cover gradient-3" style={{ height }} />
       <div className="banner">
         <div className="branding" />
-        <h1>We’re sorry to see you go...</h1>
-        {/* Drop typeform code here */}
+        <h1>We’re sorry to see you go</h1>
         <p>
-          It didn't work out this time, but you can still keep in touch!
+          Help us build the wallet <span>YOU</span> deserve by answering these two questions?
```
Comment (jagodarybacka) [inline src/pages/goodbye.tsx:109]: don't we want to use translations here?

**Comment by jagodarybacka** [comment]
I can still see the old page on uninstall 🤔

**Review by jagodarybacka — COMMENTED** [review]
Seems fine but the embed works weird - second time I opened the page it changed theme to white/blue. Also initial theme is not using the same colors as we have in figma, not sure if we can do anything about it cc @VladUXUI 

image

**Comment by kkosiorowska** [comment]
> Seems fine but the embed works weird - second time I opened the page it changed theme to white/blue. Also initial theme is not using the same colors as we have in figma, not sure if we can do anything about it cc @VladUXUI

The form is simply embedded on the page, to change the styles you need to do it through typeform. It looks like someone updated them and that's why the form is white and blue.

**Review by jagodarybacka — APPROVED** [review]
Looks better now 👍 
image

**Pull request merged**